test/android/core/javatests/src/org/xwalk/core/xwview/test/OnRequestFocusTest.java

```diff
@@ -8,6 +8,8 @@
 
 import org.chromium.base.test.util.Feature;
 
+import android.test.FlakyTest;
```

Comment:
The test you're disabling below is not flaky, it just needs to be disabled. Flaky tests are tests that randomly fail because of race conditions or because they are dependent on the timing or a certain ordering of events. In this case, we just want to skip the test, as it deterministically fails 100% of the time.